apps/condo/domains/billing/utils/serverSchema/index.js

```diff
@@ -57,21 +57,20 @@ async function registerBillingReceipts (context, data) {
  * @param context {Object}
  * @param organizationId {string}
  * @param accountNumber {string}
- * @param bic {string}
  * @param bankAccount {string}
  * @param period {string}
  * @return {Promise<*>}
  */
-const getPaymentsSum = async (context, organizationId, accountNumber, period, bic, bankAccount) => {
+// exclude routingNumber check, as it can be changed (it's only identify bank office location)
```

Comment:
I think it's better to rephrase, so it does not start with verb:

Note: (zuch) RoutingNumber is not checked, because it only identifies bank office location, which can be changed.

Reply:
this function get payments sum = so, it will stay named as getPaymentsSum ;-)

Reply:
I'm not about the function name) im about the code comment